packages/calling-stateful-client/review/beta/calling-stateful-client.api.md

```diff
@@ -115,6 +115,7 @@ export interface CallState {
     isScreenSharingOn: boolean;
     kind: CallKind;
     localVideoStreams: LocalVideoStreamState[];
+    optimalVideoCountFeature: OptimalVideoCountFeatureState;
```

Comment:
Are we expecting more related state in the OVC feature? If not just make `optimalVideoCount: number` top level

Reply:
There are plans to have the number of the videos rendered there as well. @alkwa-msft what do you think about this suggestion? Both makes sense, keeping it will help to use the new values easily in the future. But, having `optimalVideoCount` is simpler and cleaner

Reply:
if we know that there are more properties relating to OVC coming then we will want to keep it like this. If we were to top level this and then want to make a interface to wrap the state together for them this would be a breaking change to the API. I happy leaving this as is knowing that.